### PR TITLE
Add probabilistic precipitation forecasts

### PR DESCRIPTION
**Pull request by HF7weatherman**
This MR adds the probabilistic forecast of 3-hourly precipitation to the weather briefing in the "Supporting Figures" section. For an example figure see below. The briefing still generates without any problems after including these changes.

precip_enfo1

### CODE REVIEW

**Comment by HF7weatherman** [comment]
@Lucalino @sortega87 Just a quick reminder asking for your revision of this MR 😉

**Comment by sortega87** [comment]
> @Lucalino @sortega87 Just a quick reminder asking for your revision of this MR 😉

Hi @HF7weatherman! Thanks for the reminder, I had missed it somehow. Looks good to me and can be merge after addressing my title comment above. Thanks for implementing the plots :)

**Review by sortega87 — CHANGES_REQUESTED** [review]

File: src/wblib/template.qmd [hunk 1]
```diff
@@ -145,4 +145,22 @@ Summary of the weather briefing.
...
+### Probabilistic precipitation forecast
+::: {layout-nrow=2}
+![{{< meta valid_dates.015h >}}]({{< meta plots.internal.precip_enfo.015h >}}){width=42%}
+![{{< meta valid_dates.039h >}}]({{< meta plots.internal.precip_enfo.039h >}}){width=42%}
+![{{< meta valid_dates.063h >}}]({{< meta plots.internal.precip_enfo.063h >}}){width=42%}
+![{{< meta valid_dates.084h >}}]({{< meta plots.internal.precip_enfo.084h >}}){width=42%}
+:::
+
+## 
+### Horizontal sfc. wind conv. & Prob. precip. forecast
```
[the remark this answers is not in the document]
Comment (HF7weatherman) [reply src/wblib/template.qmd:160]: On this slide I would show both, surface convergence and the probabilistic precip forecast for lead times `+108h` and `+132h`. Thus, I would leave the header as is.

[the remark this answers is not in the document]
Comment (sortega87) [reply src/wblib/template.qmd:160]: Sounds good. Thanks Henning!

**Review by sortega87 — APPROVED** [review]
Looks good! Thanks Henning!

**Pull request merged**